app.rb

```diff
@@ -0,0 +1,31 @@
+require 'sinatra/base'
+require_relative './lib/chitter'
+require_relative './lib/user'
+class ChitterManager < Sinatra::Base
+  enable :sessions
+
+  before do
+    @chitter = Chitter
+    @user = User
+  end
+
+  get '/' do
+    erb :form
+  end
+
+  post '/type_peeps' do
```

Comment:
POST route should be used to change status and then redirect to a GET route to show data